.github/workflows/test.yaml

```diff
@@ -0,0 +1,32 @@
+# .github/workflows/test.yaml
+name: test
+on: push
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      RUN_AS: 1001:121
+    # Service containers to run with `container-job`
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub PostgreSQL image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+    steps:
+      - uses: actions/checkout@v2
+      - name: Test
+        uses: docker://mcreations/sbcl:2.1.1-ql-2021-04-11
```

Comment:
Should this one perhaps be `clfoundation/sbcl` ?

Reply:
Sure, I can change that. The good thing about the mcreations/sbcl image which is used here is, that it contains all of the QuickLisp distribution which is mentioned in the tag (the unpacked archives of the systems) and thus doesn't need to download anything when running. It also contains the sqlite libs so no additional package installation is needed.

The upside is a much quicker test run.

I will try the difference and report back here.

Reply:
And what I forgot: `mcreations/sbcl` contains a machinery to run the SBCL process inside the container with a specific UID and GID (env var `RUN_AS`), which is needed to allow `provider-create-migration` write access to the `t/migrations` directory during the tests.

Reply:
Checked the runtime and the permissions: both are non-issues, as the run duration is comparable and `clfoundation/sbcl` seems to run as `root` which is not a wise decision in terms of security, but removes the write permission problem.

So, I will change the image.